<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DangKiVe</class>
 <widget class="QWidget" name="DangKiVe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tblLichSuGiaHan">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>631</width>
     <height>211</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Mã Gian Hạn vé</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Mã Phương Tiện</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Biển số xe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Loại vé</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ngày gia hạn mới</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Số tiền gia hạn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGiaHan">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Gia hạn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLamMoi">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Làm mới</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ngày gia hạn</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dtNgayGiaHan">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtGiaVe">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbLoaiVe">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbMaPhuongTien">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMaGiaHan">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Mã Gia Hạn</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>40</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Loại vé</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnThoat">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Thoát</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTuKhoa">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Tìm Kiếm vé</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTimKiem">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Tìm Kiếm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
